### PR TITLE
init model and relationship

### PR DESCRIPTION
**Pull request by nguyenvandatb**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
+  gem "minitest-reporters", "1.1.9"
+  gem "guard", "2.13.0"
+  gem "guard-minitest","2.4.4"
+end
+
+group :production do
+  gem "pg", "0.18.4"
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
```
Comment (hungnh103) [inline Gemfile]: remove comment

File: app/models/lesson.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Lesson < ApplicationRecord
+  belongs_to :user
+  belongs_to :category
+  has_many :results
+  has_many :words, through: :results
+
```
Comment (hungnh103) [inline app/models/lesson.rb]: remove line

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
```
Comment (hungnh103) [inline db/seeds.rb]: remove comment in this file

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user,     only: :destroy
```
Comment (thangtx88) [inline app/controllers/users_controller.rb]: bỏ khoảng trắng

**Comment by cntt11b1** [comment]
OK

**Pull request merged**